backend/src/graphql/resolver/UserResolver.test.ts

```diff
@@ -350,15 +326,15 @@ describe('UserResolver', () => {
             data: {
               login: {
                 coinanimation: true,
-                email: 'peter@lustig.de',
-                firstName: 'Peter',
+                email: 'bibi@bloxberg.de',
+                firstName: 'Bibi',
                 hasElopage: false,
                 isAdmin: false,
                 klickTipp: {
                   newsletterState: false,
                 },
                 language: 'de',
-                lastName: 'Lustig',
+                lastName: 'Bloxberg',
                 publisherId: 1234,
```

Comment:
use the `bibiBloxberg` object value here aswell?
I am not sure what the testing strategy should be.
`If I put values in, I expect the same values back`
This would make the test independent from the data.
Or:
`If I put values in, I expect explicitly specified values back`
This would make the test detect data change and forces the developer to rethink decisions.